TrustSDK/Classes/Client/Types/TrustCommand.swift

```diff
@@ -10,11 +10,73 @@ import TrustWalletCore
 enum CommandName: String {
     case getAccounts = "sdk_get_accounts"
     case sign = "sdk_sign"
+    case signThenSend = "sdk_sign_then_send"
+}
+
+enum SignMetadataName: String {
+    case dApp = "dapp"
 }
 
 public extension TrustSDK {
+    enum SignMetadata {
+        case dApp(name: String, url: URL?)
+
+        init?(value: String) {
```

Comment:
what does code do?

Reply:
I added a generic code to encode/decode multilevel dictionaries into `[URLQueryItem]`. Please check `DictionaryTests` to understand how it works.